spec/features/anonymous_user_happy_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+feature 'Anonymous User visits the site and enters an ugly URL' do
+  scenario 'they get back a beautiful one', js: true do
+    visit root_path
+
+    fill_in 'url[original]', with: 'https://www.twitter.com'
+    click_button 'Make Beautiful'
+    sleep(0.5)
+    url = Url.find_by(original: 'https://www.twitter.com')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.